plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/auth/password/PinotPasswordAuthenticationProvider.java

```diff
@@ -39,8 +40,9 @@ public Optional<String> getAuthenticationToken()
         return authToken;
     }
 
-    private String encode(String username, String password)
+    @VisibleForTesting
```

Comment:
does this still need `visibleForTesting`? They are in different modules as far I can see.